cmd/status.go

```diff
@@ -46,8 +45,8 @@ func getStatusCmd(ctx context.Context) *cobra.Command {
 			if err != nil {
 				return err
 			}
-			ui.Dump(stdout, status)
-			return nil
+
+			return yamlPrint(stdout, status)
```

Comment:
Maybe this is a good case for wrapping errors as "could not marshal YAML:" will be quite strange error for someone calling `k6 pause` or co.

Reply:
Yeah, though to be fair, the new state isn't any worse than the previous behavior where it would have printed just the raw YAML marshaling error, without even the `could not marshal YAML:` in front of it :sweat_smile: but yeah, I may wrap these errors locally as well...